plugin-server/src/worker/vm/extensions/geoip.ts

```diff
@@ -1,12 +1,20 @@
 import { GeoIPExtension } from '@posthog/plugin-scaffold'
 
 import { Hub } from '../../../types'
-import { fetchIpLocationInternally } from '../../mmdb'
```

Comment:
Maybe lost in the diff, but did you also remove `fetchIpLocationInternally` or is it still be being used elsewhere?